appserver/tests/pom.xml

```diff
@@ -132,6 +133,7 @@
             <modules>
                 <!-- Builds compile dependencies for Ant tests executed by runtest.sh/gftest.sh -->
                 <module>appserv-tests</module>
+                <module>embedded</module>
```

Comment:
I am not sure now, but fastest doesn't run any tests. appserv-tests are there just because they produce some artifacts required by additional ant tests executed in separate stage in jenkins.

Reply:
You are right. I didn't realize that at first, I removed embedded module from the `fastest` profile with a later commit.